subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -997,6 +1022,13 @@ private boolean noDaemonArgumentGiven() {
             allArgs.add("--warning-mode=" + warningMode.toString().toLowerCase(Locale.ENGLISH));
         }
 
+        if(disableToolchainDownload) {
```

Comment:
```suggestion
        if (disableToolchainDownload) {
```

Nit

Reply:
We really need either IJ auto formatter or spotless :)